Replace RequestContext with standard dicts

sdreher opened the pull request:
* for Django 1.10 compliance

sdreher committed:
Replace RequestContext with standard dicts
* for Django 1.10 compliance

nikolas commented:
Hmmm, it looks like the upstream version may already have django 1.10 support - https://github.com/Alir3z4/django-databrowse but a release hasn't been made in a while. I'll ask this repo's owner if he can make a new release.

nikolas commented:
https://github.com/Alir3z4/django-databrowse/issues/33

nikolas commented:
Well, it looks like we have other custom changes in this package too: https://github.com/ccnmtl/django-databrowse/pull/3/files

So, I'm fine with going ahead with our custom package here as well. From the failure, it looks like the run_tests.py script here might need some attention.

nikolas committed:
Add TEMPLATES to test config

https://github.com/Alir3z4/django-databrowse/commit/ea4fe9eb4f4858d0c5a3ff60272d518ab91c5bf5

nikolas committed:
Test on django 1.10

nikolas committed:
travis: use DJANGO_VERSION instead of requirements.txt

nikolas committed:
remove unused vars from run_tests.py
